### DISCUSSION TITLE
Make a tabview like Chrome

### DISCUSSION THREAD

**Discussion by holdxen** [bdlukaa/fluent_ui#1105] [Q&A] [answered]
How can I make this area become `DragToMoveArea` in `window_manager`, just like google chrome

![Screenshot](https://github.com/user-attachments/assets/d6bd7f21-6c91-46a7-841d-4268459cedf4)

**Comment by bdlukaa** [comment, marked as answer]
You can use `TabView.stripBuilder` property to build around the strip.
